core/src/test/java/org/apache/iceberg/rest/TestRESTCatalog.java

```diff
@@ -918,8 +919,16 @@ public void testCatalogTokenRefresh() throws Exception {
             UUID.randomUUID().toString(), "user", contextCredentials, ImmutableMap.of());
 
     RESTCatalog catalog = new RESTCatalog(context, (config) -> adapter);
+    String scope = "custom_catalog_scope";
     catalog.initialize(
-        "prod", ImmutableMap.of(CatalogProperties.URI, "ignored", "credential", "catalog:secret"));
+        "prod",
+        ImmutableMap.of(
+            CatalogProperties.URI,
+            "ignored",
+            "credential",
+            "catalog:secret",
+            OAuth2Properties.SCOPE,
+            scope));
```

Comment:
Can you add a separate test case rather than repurposing an existing one?

Reply:
done